cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@ptouchton It looks like this was accidentally added?

Reply:
@theodur thanks for catching it....I thought I had reverted